nogo_config.json

```diff
@@ -140,6 +140,9 @@
     "exclude_files": {
       "vendor/": "vendor doesn't pass vet",
       "external/": "externaldoesn't pass vet"
+    },
+    "analyzer_flags": {
+      "suppress-focus-container": "true"
```

Comment:
Can this be override using a command argument?
I suspect CI is ignoring focus tests anyway, so a job definition or the CI entry point script could use a flag/argument to control it.

Reply:
I wasn't able to find a way of passing these through to `bazel`, for now I've added a hackaround in b3edf0098fb9982a8d72c6d75a9d6529f8aaecf1 that modifies the `nogo_config.json` file directly when the `KUBEVIRT_E2E_FOCUS` env variable is set.